src/load_inputs/load_generators_data.jl

```diff
@@ -501,4 +502,22 @@ function load_vre_stor_data!(inputs_gen::Dict, setup::Dict, path::AbstractString
 		inputs_gen["dfVRE_STOR"] = DataFrame()
 	end
 	summarize_errors(error_strings)
-end
\ No newline at end of file
+end
+
+
+function process_piecewisefuelusage!(inputs::Dict, scale_factor)
+	gen_in = inputs["dfGen"]
+	pwfu_num_segments = maximum(gen_in[!,:PWFU_NUM_SEGMENTS])
```

Comment:
yeah if the maximum segment is 2, for a generator with only one segment, it will force the fuel usage to equal to a linear line (fuel = slope1*vp+ intercept1), and the second segment will be fuel > 0.